irs-edc-client/src/main/java/org/eclipse/tractusx/irs/edc/client/contract/service/EdcContractAgreementService.java

```diff
@@ -68,9 +72,10 @@ public List<ContractAgreement> getContractAgreements(final String... contractAgr
     }
 
     public ContractNegotiation getContractAgreementNegotiation(final String contractAgreementId) {
+        final EndpointConfig endpoint = config.getControlplane().getEndpoint();
+        final String contractAgreements = endpoint.getContractAgreements();
         final ResponseEntity<ContractNegotiation> contractNegotiationResponseEntity = edcRestTemplate.getForEntity(
-                config.getControlplane().getEndpoint().getContractAgreements() + "/" + contractAgreementId
-                        + "/negotiation", ContractNegotiation.class);
+                endpoint.getData() + contractAgreements + "/" + contractAgreementId + "/negotiation", ContractNegotiation.class);
```

Comment:
contractAgreements is used as a variable and data not why is this used mixed mode?

Reply:
see above